src/Console/ImportFromRemoteCommand.php

```diff
@@ -47,6 +49,20 @@ protected function configure()
             'Set this flag to skip backup of local admin accounts',
             false
         );
+        $this->addOption(
+            self::OPTION_CORE_CONFIG_DATA_BACKUP,
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'Set this flag to back up the whole core_config_data table',
+            false
+        );
+        $this->addOption(
+            self::OPTION_NO_CONFIG_PATH_KEEP,
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'Set this flag to skip the backup/restore of the configured config paths',
+            false
+        );
```

Comment:
The relation between these two flags is not clear to me. Is there any relation or do they just seem to affect the same thing? Could explained it?

Reply:
I think I see where @bartoszkubicki  is coming from regarding the naming of these flags, it's slightly confusing.

I wonder if it's worth, consolidating these into a single flag which might make the command a bit easier to understand.

For instance, we could structure it like this:

--restore-config=preserve-local
This option would keep the local config table intact without importing anything from the source.

--restore-config=merge-and-override
This would handle importing the config table from the source and selectively overriding local configurations based on specific patterns.

--restore-config=skip
As the name suggests, this would skip the configuration import entirely.

and if I'm understanding it correctly, `--restore-config=merge-and-override` would be the default option.

Reply:
@bartoszkubicki @JamelleG 
I updated the command and readme to use a single flag with 3 possible values.

Reply:
Looks better, thanks

Reply:
Taking into account possible values it seems that option could be required as not using it completely does the same as option with value skip? Or maybe skip value is irrelevant and could be handled by just not adding the option?

Reply:
By default it retains the preconfigured paths, skip and preserve are the options that need to be set explicitly. I'd say it's better to keep it optional, this way the default behaviour with no options will be to
* preserve only the payment data
* keep admin accounts intact

Reply:
ok